src/fragments/lib/geo/android/escapehatch.mdx

```diff
@@ -35,7 +35,18 @@ Log.i("MyAmplifyApp", response.entries.toString())
 </Block>
 </BlockSwitcher>
 
-**API Documentation Resources**
+### Documentation Resources
 
+* [How to manage Amazon Location Service resources through console](https://docs.aws.amazon.com/location/latest/developerguide/welcome.html)
```

Comment:
Small nit: should this just link to the console? The dev guide seems to be more geared toward code based rather than the console

Reply:
I think this is good as is. There's quite a bit of _how to manage / setup resources in the console_ included in the dev guide. And since this is a general reference link vs a guided walkthrough (e.g. "Click <link to console resource page> to create resource X, then do ..."), a link to the console doesn't provide much value here.